token-swap/js/package.json

```diff
@@ -32,7 +32,7 @@
   "scripts": {
     "build": "tsc -p tsconfig.json && tsc-esm -p tsconfig.json && tsc -p tsconfig.cjs.json",
     "postbuild": "echo '{\"type\":\"commonjs\"}' > dist/cjs/package.json && echo '{\"type\":\"module\"}' > dist/esm/package.json",
-    "test": "ts-node test/main.ts",
+    "test": "mocha test",
```

Comment:
This is fine, but I'm curious if we want align on either `jest` or `mocha` for all JS libs?

Reply:
We probably want to use `jest`, but it was such a nightmare to get right in the memo directory I decided to keep it simple here. We're probably dropping all of this stuff soon anyway

Reply:
You mean the example programs?

Reply:
yeah, a lot of the unmaintained reference implementations